cmd/ebrelayer/relayer/ethereum.go

```diff
@@ -116,11 +108,12 @@ func NewEthereumSub(inBuf io.Reader, rpcURL string, cdc *codec.Codec, validatorM
 }
 
 // LoadTendermintCLIContext : loads CLI context for tendermint txs
-func LoadTendermintCLIContext(appCodec *amino.Codec, validatorAddress sdk.ValAddress, validatorName string,
-	rpcURL string, chainID string, sugaredLogger *zap.SugaredLogger) (sdkContext.CLIContext, error) {
+func LoadTendermintCLIContext(validatorAddress sdk.ValAddress, validatorName string,
+	rpcURL string, chainID string, sugaredLogger *zap.SugaredLogger) (client.Context, error) {
```

Comment:
We never use the chainID parameter, so it should be removed.

Is it correct that we never use it?  Seems like it should be used in this kind of function.

Reply:
chainID not used now. we may use it in the future as network descriptor, just keep it for now.